x-pack/plugins/case/common/api/cases/case.ts

```diff
@@ -15,12 +15,23 @@ import { CaseConnectorRt, ESCaseConnector, ConnectorPartialFieldsRt } from '../c
 // eslint-disable-next-line @kbn/eslint/no-restricted-paths
 export { ActionTypeExecutorResult } from '../../../../actions/server/types';
 
-const StatusRt = rt.union([rt.literal('open'), rt.literal('closed')]);
+const CaseStatusRt = rt.union([
+  rt.literal('open'),
+  rt.literal('in-progress'),
+  rt.literal('closed'),
+]);
+
+export const caseStatuses = CaseStatusRt.types.map((type) => type.value);
+export enum CaseStatuses {
+  open = 'open',
+  'in-progress' = 'in-progress',
```

Comment:
maybe we could make it as `in_progress`, so we don't have to use `['in-progress']` to access it?

Reply:
We have to match the alert's statuses so we can sync cases and alerts seamlessly.